onadata/apps/api/tools.py

```diff
@@ -652,10 +651,14 @@ def get_xform_users(xform):
     :return:
     """
     data = {}
+    org_members = None
```

Comment:
Could this be an empty list that you simply iterate over it without the if check in line 659?